Deploy/stacks/Chile/README.md

```diff
@@ -0,0 +1,141 @@
+# Cross-domain representation of Chilean energy-related information
+
+Last update: 4th of July 2024. 
+
+        * Reproduced by Feroz on Tuesday 16 of January 2024 (from Dropbox). 
+        * Reproduced by Andrea on 03 July 2024 from dropbox and GitHub. 
+        * Reproduced again by Feroz on 05 July 2024 in a different machine.
+
+## Prerequisites
+You need to have Windows, WSL and Ubuntu, Visual Studio Code, Docker, Java 11 all installed in the computer, 
+and perhaps Git, Tortoise, and TomCat. Good luck.
+
+In the case of Linux, you need Visual Studio Code, Docker, Java 11 all installed in the computer, and perhaps Git, and TomCat. Good luck.
+
+## Instructions
+
+These instructions should allow to see a cross-domain representation of Chilean information.
+
+
+### Step 1. Clone TheWorldAvatar, which is a GitHub repository available at https://github.com/cambridge-cares/TheWorldAvatar.git.
+### Step 2. Populate following folders (the best way to do it is using Visual Studio Code to transfer the files in the correct folder):
+
+* TheWorldAvatar/Deploy/stacks/dynamic/stack-data-uploader/inputs/config/
+* TheWorldAvatar/Deploy/stacks/dynamic/stack-data-uploader/inputs/data/
+* TheWorldAvatar/Deploy/stacks/dynamic/stack-manager/inputs/config/
+* TheWorldAvatar/Deploy/stacks/dynamic/stack-manager/inputs/config/services/  (this one allows to access the geoserver with all the permissions)
+* TheWorldAvatar/Deploy/stacks/dynamic/stack-manager/inputs/data/webspace/
+* TheWorldAvatar/Deploy/stacks/dynamic/stack-manager/inputs/data/webspace/data/ (here must go all the icons, if you copy all the webspace folder, it is already included)
+
+With the data files and folders found in the dropbox archive folder, which can be requested to the group.
+
+With the configuration files found in GitHub at:
+
+* TheWorldAvatar/Deploy/stacks/Chile/ (in the repository at GitHub)
+
+
+The folders to populate will be found in TheWorldAvatar/Deploy/stacks/dynamic/
+
+Go to the following folder:
+
+* TheWorldAvatar/Deploy/stacks/dynamic/stack-manager/inputs/secrets/    (this is were you upload your passwords for geoserver, postgis and mapbox)
```

Comment:
Replace "(this is were you upload your passwords for geoserver, postgis and mapbox)" with "(this is where you provide your passwords for GeoServer and PostGIS, and API Key for Mapbox)"

Reply:
done